src/c.mli

```diff
@@ -0,0 +1,45 @@
+open Stdune
+
+module Kind : sig
+  type t =
+    | C
+    | Cxx
+
+  val pp : t Fmt.t
+
+  val split_fn : string -> (string * t) option
+
+  val possible_fns : t -> string -> string list
```

Comment:
It seems worth documenting that the argument should be a filename without the extension

Reply:
done